### PR TITLE
Docs: update test statistics page with example responses

### PR DESCRIPTION
**Pull request by aldenpeterson-wf**
Provide an example in the docs based on  [this Stack Overflow question](https://stackoverflow.com/questions/49197829/locust-test-statistics-meta-data/49198304#49198304).

### CODE REVIEW

File: docs/retrieving-stats.rst [hunk 1]
```diff
@@ -9,13 +9,31 @@ First, when running Locust with the web UI, you can retrieve CSV files under the
...
 You can also customize how frequently this is written if you desire faster (or slower) writing::
 
     import locust.stats
     locust.stats.CSV_STATS_INTERVAL_SEC = 5 # default is 2 seconds
+
+This data will look like::
+
+    $cat example_distribution.csv
+    "Name","# requests","50%","66%","75%","80%","90%","95%","98%","99%","100%"
+    "GET /",31,4,4,4,4,4,4,4,4,4
```
Comment (cgoldberg) [inline docs/retrieving-stats.rst]: isn't the "Method" field header missing here?

File: docs/retrieving-stats.rst [hunk 2]
```diff
@@ -9,13 +9,31 @@ First, when running Locust with the web UI, you can retrieve CSV files under the
 Secondly, you can run Locust with a flag which will periodically save the CSV file. This is particularly useful
 if you plan on running Locust in an automated way with the ``--no-web`` flag::
 
-    locust -f locust_files/my_locust_file.py --csv=foobar --no-web -n10 -t10m
+    locust -f examples/basic.py --csv=example --no-web -n10 -t10m
```
Comment (cgoldberg) [inline docs/retrieving-stats.rst]: is `example` the full name of the csv file?  Might want to name it `example.csv`

File: docs/retrieving-stats.rst [hunk 3]
```diff
@@ -9,13 +9,31 @@ First, when running Locust with the web UI, you can retrieve CSV files under the
...
 
 or for v0.8 (where the ``-t`` option isn't available)::
 
-    locust -f locust_files/my_locust_file.py --csv=foobar --no-web -n10 -c10
+    locust -f examples/basic.py --csv=example --no-web -n10 -c10
```
Comment (cgoldberg) [inline docs/retrieving-stats.rst]: same as above

File: docs/retrieving-stats.rst (hunk 1 above)
Comment (aldenpeterson-wf) [reply docs/retrieving-stats.rst]: I'm not sure, it's identical to what is generated by locust for the stats pages currently.

File: docs/retrieving-stats.rst (hunk 2 above)
Comment (aldenpeterson-wf) [reply docs/retrieving-stats.rst]: it gets prepended with the type of file, so providing `example` results in `example_distribution.csv` and `example_requests.csv`

Comment (cgoldberg) [reply docs/retrieving-stats.rst]: gotcha.  that's confusing.  maybe note that in the docs?

File: docs/retrieving-stats.rst (hunk 1 above)
Comment (cgoldberg) [reply docs/retrieving-stats.rst]: oh I see.  Method and Name are merged into a "Name" column.  This seems like a bug to me (though unrelated to your PR

File: docs/retrieving-stats.rst (hunk 2 above)
Comment (aldenpeterson-wf) [reply docs/retrieving-stats.rst]: I added clarification there

File: docs/retrieving-stats.rst [hunk 4]
```diff
@@ -6,16 +6,37 @@ You may wish to consume your Locust results via a CSV file. In this case, there
...
+Secondly, you can run Locust with a flag which will periodically save two CSV files. This is particularly useful
 if you plan on running Locust in an automated way with the ``--no-web`` flag::
 
-    locust -f locust_files/my_locust_file.py --csv=foobar --no-web -n10 -t10m
+    locust -f examples/basic.py --csv=example --no-web -n10 -t10m
 
 or for v0.8 (where the ``-t`` option isn't available)::
 
-    locust -f locust_files/my_locust_file.py --csv=foobar --no-web -n10 -c10
+    locust -f examples/basic.py --csv=example --no-web -n10 -c10
```
Comment (karol-brejna-i) [inline docs/retrieving-stats.rst]: is `-n` switch working? If not (which I suspect) maybe we should either remove it from the example or reintroduce it to the code.

Comment (aldenpeterson-wf) [reply docs/retrieving-stats.rst]: good catch, will update - it worked when I _originally_ wrote this functionality :)

Comment (cgoldberg) [reply docs/retrieving-stats.rst]: @aldenpeterson-wf can you update ^^ ?  then let's get this merged.

**Pull request merged**